src/github.com/matrix-org/smallifier/smallifier/smallifier.go

```diff
@@ -89,6 +151,13 @@ func (s *Smallifier) CreateHandler(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	if s.lengthLimit > 0 && len(jsonReq.LongURL) > s.lengthLimit {
+		log.WithField("url", jsonReq.LongURL).Error("Refusing to linkify long link")
+		w.WriteHeader(400)
+		io.WriteString(w, fmt.Sprintf(`{"error": "Links must be shorted than %d runes"}`, s.lengthLimit))
```

Comment:
I'd prefer that we used "code-point" rather than "rune" in an error message since "rune" is a golang specific term for "code-point"